pkg/helpers/azure/blob.go

```diff
@@ -0,0 +1,27 @@
+package azure
```

Comment:
I was not sure where to put that (new `helpers/azure` helpers), azure SDK is very weird.
We cannot reuse default auth mechanism than the one used by the `az` CLI, because it seems that this logic is done in the CLI itself not in the SDK.

I really don't like the `helper` pkg approach, it often became an big hold-all mess and prevent us to do a really good separation of concerns.

Any thoughts ?

Reply:
That file does not makes sense since we now use flags